shared/fs/folder/container.js

```diff
@@ -5,13 +5,21 @@ import Files from '.'
 import * as Types from '../../constants/types/fs'
 import * as Constants from '../../constants/fs'
 import SecurityPrefsPromptingHoc from '../common/security-prefs-prompting-hoc'
+import * as FsGen from '../../actions/fs-gen'
 
 const mapStateToProps = (state, {path}) => ({
+  _pathItems: state.fs.pathItems,
   _tlfs: state.fs.tlfs,
   _username: state.config.username,
   sortSetting: state.fs.pathUserSettings.get(path, Constants.makePathUserSetting()).get('sort'),
 })
 
+const mapDispatchToProps = dispatch => ({
+  onAttach: (parentPath: Types.Path, paths: Array<string>) => {
+    paths.forEach(localPath => dispatch(FsGen.createUpload({localPath, parentPath})))
+  },
```

Comment:
You could just make `onAttach` accept a single `paths` param here since `path` is already available from `ownProps` here. Then you don't need to make an `onDrop` [here](https://github.com/keybase/client/pull/15152/files#diff-6df9d6db4b7ccf7cbc83bcd2ba7ee713R26) anymore.

Reply:
Done.